client/my-sites/post-selector/docs/example.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import PostSelector from '../';
+
+export default React.createClass( {
+	displayName: 'PostSelector',
+
+	mixins: [ PureRenderMixin ],
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/app-components/post-selector">Post Selector</a>
+				</h2>
+				<div style={ { width: 300 } }>
+					<PostSelector
```

Comment:
>Should we use your primary site instead?

Yeah, I like this idea. Updated the example in the rebased 70ef26e2143ccaa5147694cc629c5d6f5df932f4.